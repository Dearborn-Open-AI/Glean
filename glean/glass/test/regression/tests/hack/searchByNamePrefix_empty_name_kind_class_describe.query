action: searchByNamePrefix
args: {"context": {"repo_name": "test", "language": 3, "kinds": [7]}, "name": "", "detailedResults": true, "ignoreCase": false }
