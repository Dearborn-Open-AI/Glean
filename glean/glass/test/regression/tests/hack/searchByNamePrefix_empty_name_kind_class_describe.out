[
    "@generated",
    {
        "symbolDetails": [
            {
                "annotations": [],
                "comments": [],
                "kind": 7,
                "location": {
                    "filepath": "www/Namespace.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 12,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "NS",
                    "localName": "NamespaceClass"
                },
                "repo_hash": "testhash",
                "sym": "test/php/NS/NamespaceClass"
            },
            {
                "annotations": [],
                "comments": [],
                "kind": 7,
                "location": {
                    "filepath": "www/RefClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 32
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "RefClass"
                },
                "repo_hash": "testhash",
                "sym": "test/php/RefClass"
            },
            {
                "annotations": [
                    {
                        "name": "TestAnnotation",
                        "source": "TestAnnotation(\"Text\")"
                    }
                ],
                "comments": [
                    {
                        "filepath": "www/SourceClass.php",
                        "range": {
                            "columnBegin": 1,
                            "columnEnd": 4,
                            "lineBegin": 10,
                            "lineEnd": 12
                        },
                        "repository": "test"
                    }
                ],
                "kind": 7,
                "location": {
                    "filepath": "www/SourceClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 13,
                        "lineEnd": 25
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "SourceClass"
                },
                "repo_hash": "testhash",
                "sym": "test/php/SourceClass"
            },
            {
                "annotations": [],
                "comments": [],
                "kind": 7,
                "location": {
                    "filepath": "www/SourceClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 38,
                        "lineBegin": 27,
                        "lineEnd": 27
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "SubClass"
                },
                "repo_hash": "testhash",
                "sym": "test/php/SubClass"
            },
            {
                "annotations": [],
                "comments": [],
                "kind": 7,
                "location": {
                    "filepath": "www/SuperClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 10,
                        "lineEnd": 17
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "SuperClass"
                },
                "repo_hash": "testhash",
                "sym": "test/php/SuperClass"
            },
            {
                "annotations": [],
                "comments": [],
                "kind": 7,
                "location": {
                    "filepath": "www/SourceClass.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 29,
                        "lineEnd": 31
                    },
                    "repository": "test"
                },
                "name": {
                    "container": "",
                    "localName": "TestAnnotation"
                },
                "repo_hash": "testhash",
                "sym": "test/php/TestAnnotation"
            }
        ],
        "symbols": [
            "test/php/NS/NamespaceClass",
            "test/php/RefClass",
            "test/php/SourceClass",
            "test/php/SubClass",
            "test/php/SuperClass",
            "test/php/TestAnnotation"
        ]
    }
]